internal/runtimehandlerhooks/runtime_handler_hooks.go

```diff
@@ -44,3 +54,23 @@ func highPerformanceAnnotationsSpecified(annotations map[string]string) bool {
 	}
 	return false
 }
+
+func cpuLoadBalancingAllowed(config *libconfig.Config) bool {
+	cpuLoadBalancingEnabledOnce.Do(func() {
+		for _, runtime := range config.Runtimes {
+			for _, ann := range runtime.AllowedAnnotations {
+				if ann == crioann.CPULoadBalancingAnnotation {
+					cpuLoadBalancingEnabled = true
```

Comment:
The name is a bit confusing, should it be something like

```suggestion
					cpuLoadBalancingControlEnabled = true
```

Reply:
I went for `cpuLoadBalancingAllowedAnywhere` does that clarify?